Example/Podfile

```diff
@@ -1,4 +1,4 @@
-platform :"ios", "8.0"
+platform :"ios", "10.0"
```

Comment:
Any reason for the bump? RxSwift still supports iOS 8 and up ... so not sure why the bump is needed

Reply:
My mistake!